searches/double_linear_search.py

```diff
@@ -0,0 +1,59 @@
+"""Implementation of a double-linear-search,
+which iterates through the array from both sides: start and end.
+
+Change ARRAY_LENGTH to a number you want to generate an example array,
+which goes up to the number you define:
+ARRAY_LENGTH = 50 --> [0, 1, 2, ..., 49, 50]
+"""
+
+# change for bigger/smaller arrays
+ARRAY_LENGTH = 100
+
+
+def double_linear_search(array, x):

```

Comment:
```suggestion
def double_linear_search(array: list, search_term: int) -> int:
```
Let’s find a more self documenting variable name than x.  Using Python type hints as discussed in CONTRIBUTING.md